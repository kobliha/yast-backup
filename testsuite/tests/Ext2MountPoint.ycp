// Backup module - testsuite
//
// Ext2MountPoint function tests
//
// $Id$
//

{

include "backup/functions.ycp";
include "testsuite.ycp";

map READ_mounts = $[ "proc" : $[ "mounts" :
[
$["file":"/", "freq":0, "mntops":"rw", "passno":0,"spec":"/dev/root", "vfstype":"reiserfs"],
$["file":"/proc", "freq":0, "mntops":"rw", "passno":0, "spec":"proc", "vfstype":"proc"],
$["file":"/dev/pts", "freq":0, "mntops":"rw", "passno":0, "spec":"devpts", "vfstype":"devpts"],
$["file":"/boot", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/hda1", "vfstype":"ext2"],
$["file":"/local", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/hda4", "vfstype":"ext2"]
]],

"etc" : $[ "fstab" :
[
$["file":"/media/cdrom", "freq":0, "mntops":"ro,noauto,user,exec", "passno":0, "spec":"/dev/cdrom", "vfstype":"auto"],
$["file":"/unmounted", "freq":0, "mntops":"defaults", "passno":0, "spec":"/dev/hdb1", "vfstype":"ext2"],
]

]];


TEST( ``(Ext2MountPoint("/dev/hda1")), [ $[], $[], $[] ], nil );
TEST( ``(Ext2MountPoint("/dev/hda1")), [ READ_mounts, $[], $[] ], nil );
TEST( ``(Ext2MountPoint("/dev/hdb1")), [ READ_mounts, $[], $[] ], nil );
TEST( ``(Ext2MountPoint("/dev/hdd5")), [ READ_mounts, $[], $[] ], nil );

}

