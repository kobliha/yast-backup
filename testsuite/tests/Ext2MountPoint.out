Read	.proc.mounts nil
Read	.etc.fstab nil
Return	nil
Read	.proc.mounts [$["file":"/", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/root", "vfstype":"reiserfs"], $["file":"/proc", "freq":0, "mntops":"rw", "passno":0, "spec":"proc", "vfstype":"proc"], $["file":"/dev/pts", "freq":0, "mntops":"rw", "passno":0, "spec":"devpts", "vfstype":"devpts"], $["file":"/boot", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/hda1", "vfstype":"ext2"], $["file":"/local", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/hda4", "vfstype":"ext2"]]
Return	/boot
Read	.proc.mounts [$["file":"/", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/root", "vfstype":"reiserfs"], $["file":"/proc", "freq":0, "mntops":"rw", "passno":0, "spec":"proc", "vfstype":"proc"], $["file":"/dev/pts", "freq":0, "mntops":"rw", "passno":0, "spec":"devpts", "vfstype":"devpts"], $["file":"/boot", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/hda1", "vfstype":"ext2"], $["file":"/local", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/hda4", "vfstype":"ext2"]]
Read	.etc.fstab [$["file":"/media/cdrom", "freq":0, "mntops":"ro,noauto,user,exec", "passno":0, "spec":"/dev/cdrom", "vfstype":"auto"], $["file":"/unmounted", "freq":0, "mntops":"defaults", "passno":0, "spec":"/dev/hdb1", "vfstype":"ext2"]]
Return	/unmounted
Read	.proc.mounts [$["file":"/", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/root", "vfstype":"reiserfs"], $["file":"/proc", "freq":0, "mntops":"rw", "passno":0, "spec":"proc", "vfstype":"proc"], $["file":"/dev/pts", "freq":0, "mntops":"rw", "passno":0, "spec":"devpts", "vfstype":"devpts"], $["file":"/boot", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/hda1", "vfstype":"ext2"], $["file":"/local", "freq":0, "mntops":"rw", "passno":0, "spec":"/dev/hda4", "vfstype":"ext2"]]
Read	.etc.fstab [$["file":"/media/cdrom", "freq":0, "mntops":"ro,noauto,user,exec", "passno":0, "spec":"/dev/cdrom", "vfstype":"auto"], $["file":"/unmounted", "freq":0, "mntops":"defaults", "passno":0, "spec":"/dev/hdb1", "vfstype":"ext2"]]
Return	nil
